src/Common/Auth/OpenIDConnect/Repositories/ClientRepository.php

```diff
@@ -12,7 +12,7 @@
 
 namespace OpenEMR\Common\Auth\OpenIDConnect\Repositories;
 
-use http\Exception\RuntimeException;
+use \RuntimeException;
```

Comment:
@adunsulag , this is giving a styling error since it has the leading slash in it. Can I just remove this line entirely since it's just called via `\RuntimeException` below?

Reply:
removed it and testing well :) 
gonna bring in this PR